gravity.sh

```diff
@@ -116,14 +119,27 @@ database_table_from_file() {
     declare -i rowid
     rowid=1
     # Read file line by line
-    grep -v '^ *#' < "${source}" | while IFS= read -r domain
-    do
-      # Only add non-empty lines
-      if [[ ! -z "${domain}" ]]; then
-        echo "${rowid},\"${domain}\",1,${timestamp},${timestamp},\"Migrated from ${source}\"" >> "${tmpFile}"
-        rowid+=1
-      fi
-    done
+    if [[ "${table}" == "auditlist" ]]; then
```

Comment:
This if statement can be moved closer to where the difference is in the two branches (when writing to `tmpFile`).

Reply:
My idea was to wrap the `if` around the `do` loop to have it only evaluated once. Otherwise, the same `if` might need to be evaluated hundreds of times if it is checked withing the `do` loop.

Reply:
That is fine, it is worth not duplicating the code. It is a very simple check and premature optimization should generally be avoided.